### DISCUSSION TITLE
Auto Save Using Custom Session Name

### DISCUSSION THREAD

**Discussion by kenreeser** [sienori/Tab-Session-Manager#946] [Ideas]
It would be extremely helpful to be able to use the auto save features (periodic, window close, & browser exit) and have my custom session names used. Currently, TSM creates new names based upon the type of auto save action. This is a great feature when a custom name has not been provided to sessions.

**Comment by Bryan-Longworth** [comment]
I'm having the same problem. I have to constantly deleted new sessions that are created while the existing session isn't updated with new tabs added. I have used Simple Tab Groups which does successfully update existing sessions or groups with new or removed tabs, but I'm trying Tab Session Manager because STG opens every tab when I open Firefox even though I only intend to open one tab group. It also temporarily adds a closed group to another open group, and if you don't catch that and close that group, it will have the tabs in that group plus the previously closed group.
